### DISCUSSION TITLE
Battery shows "battery charging" while it is discharging although I have used the *-1 "trick"

### DISCUSSION THREAD

**Discussion by Sobr87** [marq24/ha-evcc#30] [Q&A]
Hi,
I have used this line for the battery (copy - paste) from the example --> jq: .state|tonumber * -1 # this does the trick to invert the sensor value for evcc

But EVCC shows "Batterie laden" while discharging.

What is wrong in my code:
  - name: bat
    type: custom
    power:
      source: http
      uri: http:// [my-HA-INSTANCE] :8123/api/states/sensor.battery_power
      method: GET
      headers:
        - Authorization: Bearer [my token]
      insecure: true
      jq: .state|tonumber * -1 # this does the trick to invert the sensor value for evcc
      timeout: 2s

I have also tried to delete the " * -1 " but nothing was changed, I believe there is no effect of the *-1 in this line IN MY CODE.
I have also restarted the HA after every change.
How can I make the *-1 work?

Thank you all in advance!

**Comment by marq24** [comment]
remove `# this does the trick to invert the sensor value for evcc` ?!

**Reply by Sobr87** [reply]
still not working :(
I have tried *-1 to an other sensor and it was working, it seems that *-1 has no effect only to the battery sensor

**Reply by marq24** [reply]
I am sorry - but then I am clueless - since this is IMHO a 100% evcc issue (and I only provided some example docu, how it worked for me in my repo)

**Reply by Sobr87** [reply]
Thank you for your support
Now sun is shining and its easier for me to understand the system, because all sensors are working now. I think everything in correct in your description. Now I have deleted the *-1 and it seems to work, I think my inverter provides already the correct value for evcc sothat *-1 is not needed.
Thank you so much for your work, now evcc seems to work also with my inverter, I am so happy that I have found your example!!!! :):):)
